### ISSUE TITLE
Create Daily Journal Note Command Renamed

### ISSUE THREAD

**Issue by gnomesoup** [dendronhq/dendron#333]
**Describe the bug**
In Dendron v0.14.0, the command ID for "Create Daily Journal Note" was renamed to not include the `u`. 

`dendron.createDailyJornalNote`

This has broken my custom keybindings for the command. If the change was intentional, I cannot find documentation on it

**To Reproduce**
1. Go to 'keybindings.json' settings file
2. Create a new binding 
3. Start entering a command `dendron.createDailyJournalNote`
4. See the mis-named command ID in the autocomplete list.

**Expected behavior**
I expected the command to maintain the original name `dedron.createDailyJournalNote`

**Screenshots**
![image](https://user-images.githubusercontent.com/6854326/98470040-c350b200-21a8-11eb-8e9f-a290b38957e5.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 (Windows_NT x64 10.0.18363)
 - Browser: Chrome f83.0.4103.122
 - Visual Studio Code: v1.51.0

**Additional context**

** Dendron Log file**
Please attach the output of `>Dev:Dendron: Open Logs` here
```
{"level":30,"time":1604848769776,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"Logger:configure","msg":"exit","log_level":"info"}
{"level":30,"time":1604848769858,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","stage":"dev","isDebug":false,"logPath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\logs\\20201108T091342\\exthost1\\dendron.dendron","extensionPath":"c:\\Users\\mpfammatter\\.vscode\\extensions\\dendron.dendron-0.14.0","extensionUri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\.vscode\\extensions\\dendron.dendron-0.14.0","_sep":1,"path":"/c:/Users/mpfammatter/.vscode/extensions/dendron.dendron-0.14.0","scheme":"file"},"storagePath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\workspaceStorage\\e058aba0fd706605383dc5bf7ee2f0bc\\dendron.dendron"}
{"level":30,"time":1604848769860,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"DendronWorkspace","msg":"initialized"}
{"level":30,"time":1604848769860,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","installedGlobalVersion":"0.14.0","migratedGlobalVersion":"0.14.0","previousGlobalVersion":"0.13.5","previousWsVersion":"0.14.0","platform":"win32","extensions":[{"id":"dendron.dendron-paste-image","version":"1.0.4","active":false},{"id":"equinusocio.vsc-material-theme","version":"33.0.0","active":false},{"id":"dendron.dendron-markdown-shortcuts","version":"0.12.1","active":false},{"id":"dendron.dendron-markdown-preview-enhanced","version":"0.10.2","active":false},{"id":"dendron.dendron-markdown-links","version":"0.6.8","active":false},{"id":"dendron.dendron-markdown-notes","version":"0.0.17","active":false},{"id":"github.github-vscode-theme","version":"1.1.5","active":false}],"workspace":"c:\\Users\\mpfammatter\\Dendron\\vault"}
{"level":30,"time":1604848769861,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","msg":"wsActive","lspSupport":true}
{"level":30,"time":1604848769862,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","msg":"start with lsp support"}
{"level":30,"time":1604848769887,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron.server","ctx":"launch","msg":"exit","port":57368,"LOG_DST":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\logs\\20201108T091342\\exthost1\\dendron.dendron\\dendron.server.log","lvl":"info"}
{"level":30,"time":1604848769887,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","msg":"post-start-server","port":57368}
{"level":30,"time":1604848769920,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"ReloadIndex.execute","msg":"enter"}
{"level":30,"time":1604848777697,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"ReloadIndex.execute","msg":"exit"}
{"level":30,"time":1604848777697,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"reloadWorkspace","msg":"post-ws.reloadWorkspace"}
{"level":30,"time":1604848777698,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"reloadWorkspace","msg":"exit"}
{"level":30,"time":1604848777699,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"postReloadWorkspace","msg":"same wsVersion"}
{"level":30,"time":1604848777699,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"postReloadWorkspace","msg":"exit"}
{"level":30,"time":1604848777699,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree"}
{"level":30,"time":1604848777812,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"activateWorkspace","stage":"dev","msg":"enter"}
{"level":30,"time":1604848777816,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"_activate","msg":"fin startClient"}
{"level":30,"time":1604848777816,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.14.0","previousVersion":"0.14.0"}
{"level":30,"time":1604848777816,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"not first time install"}
{"level":30,"time":1604848778284,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"TreeView:getChildren","msg":"reconstructing tree"}
{"level":30,"time":1604848778511,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"reloadWorkspace","msg":"gotConfig","config":{"folders":[{"path":"vault"}],"settings":{"dendron.rootDir":".","dendron.dailyJournalDomain":"daily","dendron.defaultJournalDateFormat":"Y.MM.DD","dendron.defaultJournalName":"journal","dendron.useExperimentalLSPSupport":true,"editor.minimap.enabled":false,"files.autoSave":"onFocusChange","workbench.colorTheme":"Community Material Theme Darker","pasteImage.path":"${currentFileDir}/assets/images","pasteImage.prefix":"/","markdown-preview-enhanced.enableWikiLinkSyntax":true,"markdown-preview-enhanced.wikiLinkFileExtension":".md","markdown-preview-enhanced.enableScriptExecution":true,"vscodeMarkdownNotes.noteCompletionConvention":"noExtension","vscodeMarkdownNotes.slugifyCharacter":"NONE","editor.snippetSuggestions":"inline","editor.suggest.snippetsPreventQuickSuggestions":false,"editor.suggest.showSnippets":true,"editor.tabCompletion":"on","gitdoc.enabled":true,"gitdoc.autoPush":"onCommit","todo-tree.tree.scanMode":"workspace","[markdown]":{"editor.wordWrap":"on","editor.quickSuggestions":{"other":true,"comments":false,"strings":false},"editor.tabSize":2,"editor.wordBasedSuggestions":false},"files.associations":{"*.yaml":"home-assistant"}},"extensions":{"recommendations":["dendron.dendron-paste-image","equinusocio.vsc-material-theme","dendron.dendron-markdown-shortcuts","dendron.dendron-markdown-preview-enhanced","dendron.dendron-markdown-links","dendron.dendron-markdown-notes"],"unwantedRecommendations":["shd101wyy.markdown-preview-enhanced","kortina.vscode-markdown-notes","mushan.vscode-paste-image"]}}}
{"level":30,"time":1604848801029,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","_sep":1,"external":"vscode-userdata:/c%3A/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","path":"/C:/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","scheme":"vscode-userdata"}}
{"level":30,"time":1604848861493,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","_sep":1,"external":"vscode-userdata:/c%3A/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","path":"/C:/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","scheme":"vscode-userdata"}}
{"level":30,"time":1604848883228,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604848903317,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604848904973,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604848908751,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604848925450,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\Dendron\\vault\\daily.journal.2020.11.07.md","_sep":1,"external":"file:///c%3A/Users/mpfammatter/Dendron/vault/daily.journal.2020.11.07.md","path":"/c:/Users/mpfammatter/Dendron/vault/daily.journal.2020.11.07.md","scheme":"file"}}
{"level":30,"time":1604848925454,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604849459356,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\settings.json","_sep":1,"external":"vscode-userdata:/c%3A/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","path":"/C:/Users/mpfammatter/AppData/Roaming/Code/User/settings.json","scheme":"vscode-userdata"}}
{"level":30,"time":1604849638114,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WindowWatcher:onDidChangeActiveTextEditor","msg":"enter","uri":{"$mid":1,"fsPath":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\keybindings.json","_sep":1,"external":"vscode-userdata:/c%3A/Users/mpfammatter/AppData/Roaming/Code/User/keybindings.json","path":"/C:/Users/mpfammatter/AppData/Roaming/Code/User/keybindings.json","scheme":"vscode-userdata"}}
{"level":30,"time":1604849687206,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\keybindings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604849739827,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\keybindings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604850959721,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\keybindings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604851423941,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"WorkspaceWatcher:onWillSaveTextDocument","url":"c:\\Users\\mpfammatter\\AppData\\Roaming\\Code\\User\\keybindings.json","reason":3,"msg":"enter"}
{"level":30,"time":1604851455275,"pid":13436,"hostname":"LAPTOP-0HEG6TL4","name":"dendron","ctx":"execute","opts":{}}
```

**Comment by kevinslin** [comment]
You're right. This is a regression. Will be fixed in our upcoming release!

**Comment by kevinslin** [comment]
Fixed in 0.15 release. Thanks again for the heads up!
